src/components/attribution/AttributionDirective.js

```diff
@@ -35,8 +40,60 @@ goog.provide('ga_attribution_directive');
                 '</div></div>'
           });
         }
+        var update = function(element, layers) {
+          var attrs = {}, list = [], is3dActive = scope.is3dActive();
+          if (is3dActive) {
+            layers = [scope.ol3d.getCesiumScene().terrainProvider].
+                concat(layers);
+          }
+          layers.forEach(function(layer) {
+            var key = gaAttribution.getTextFromLayer(layer, is3dActive);
+            if (!attrs[key]) {
+              var attrib = gaAttribution.getHtmlFromLayer(layer, is3dActive);
+              if (attrib) {
+                attrs[key] = attrib;
+                list.push(attrs[key]);
+              }
+            }
+          });
+          var text = list.join(', ');
+          element.html((text) ? $translate.instant('copyright_data') + text :
```

Comment:
You could remove the brackets surrounding `text`, unless this is a convention you guys have.